### ISSUE TITLE
Installation error on Fedora 41 Beta

### ISSUE THREAD

**Issue by itaberas** [tailscale/tailscale#13828]
### What is the issue?

When running the installation script on Fedora 41 Beta "curl -fsSL https://tailscale.com/install.sh | sh && sudo tailscale up --auth-key=tskey-auth-xxxxxxxxxxxxxxxx", it returns this error:

\+ sudo dnf config-manager --add-repo https://pkgs.tailscale.com/stable/fedora//tailscale.repo
Unknown argument "--add-repo" for command "config-manager". Add "--help" for more information about the arguments.

I temporarily used "curl -fsSL https://tailscale.com/install.sh | sed 's/dnf config-manager/dnf-3 config-manager/g' | sh && sudo tailscale up --auth-key=tskey-auth-xxxxxxxxxxxxxxxx"

### Steps to reproduce

Run the installation command on Fedora 41 Beta

### Are there any recent changes that introduced the issue?

Upgrade to dnf5 on Fedora

### OS

Linux

### OS version

Fedora 41 Beta

### Tailscale version

1.76.0

**Comment by andrew-d** [comment]
@itaberas Hi there, and thanks for the report! I just merged #13844, which adds support for DNF v5, and thus Fedora 41 Beta. I tested this in a VM, but if you have a minute to confirm that this works for you, that would be appreciated.

**Comment by ajayramaswamy** [comment]
the format of  dnf5 add repo that works for me is
```
sudo dnf config-manager addrepo --from-repofile="https://pkgs.tailscale.com/stable/fedora/tailscale.repo"
```

**Comment by itaberas** [comment]
> @itaberas Hi there, and thanks for the report! I just merged #13844, which adds support for DNF v5, and thus Fedora 41 Beta. I tested this in a VM, but if you have a minute to confirm that this works for you, that would be appreciated.

I tested the installation on my Fedora 41, and it has now installed correctly. Thank you for the support.

**Comment by andrew-d** [comment]
Awesome! I'm going to close this now; feel free to re-open if you see any additional installation issues.
